diff --git a/cmake/01-core.cmake b/cmake/01-core.cmake
index fe0032e5..b50e625d 100644
--- a/cmake/01-core.cmake
+++ b/cmake/01-core.cmake
@@ -15,7 +15,7 @@ option(BUILD_POLYBAR "Build the main polybar executable" ${DEFAULT_ON})
 option(BUILD_POLYBAR_MSG "Build polybar-msg" ${DEFAULT_ON})
 option(BUILD_TESTS "Build testsuite" OFF)
 option(BUILD_DOC "Build documentation" ${DEFAULT_ON})
-option(BUILD_CONFIG "Generate sample configuration" ${DEFAULT_ON})
+option(BUILD_CONFIG "Generate sample configuration" OFF)
 option(BUILD_SHELL "Generate shell completion files" ${DEFAULT_ON})
 
 include(CMakeDependentOption)
